cmp/cmpopts/example_test.go

```diff
@@ -0,0 +1,125 @@
+package cmpopts_test
+
+import (
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/google/go-cmp/cmp/internal/flags"
+)
+
+func init() {
+	flags.Deterministic = true
+}
+
+// Use IgnoreFields to ignore fields on a type when comparing.
+// Provide an interface of the type, and the field names to ignore.
```

Comment:
Nit: drop the comma before the "and" as "the field names to ignore" is not a complete sentence.

Reply:
Done